### ISSUE TITLE
Acquire census data

### ISSUE THREAD

**Issue by futurechris** [hackoregon/emergency-response#38]
Changing this to just be the census data ticket.

 - [ ] Where do we want to store this?
 - [ ] Store in raw form or analyzed?

**Comment by sky-t** [comment]
Update: the following tables of census data now exist in the db: 

* census_household_income
* census_household_language
* census_households_65plus
* census_housing_tenure
* census_median_household_income
* census_race
* census_total_population

The data dictionary has been updated with descriptions. At this point, the only other tables I plan to add data from are:

* B27010 - TYPES OF HEALTH INSURANCE COVERAGE BY AGE
* B17021 - POVERTY STATUS OF INDIVIDUALS IN THE PAST 12 MONTHS BY LIVING ARRANGEMENT
* B22010 - RECEIPT OF FOOD STAMPS/SNAP IN THE PAST 12 MONTHS BY DISABILITY STATUS FOR HOUSEHOLDS
* B15003 - EDUCATIONAL ATTAINMENT FOR THE POPULATION 25 YEARS AND OVER
* B07201 - GEOGRAPHICAL MOBILITY IN THE PAST YEAR FOR CURRENT RESIDENCE--METROPOLITAN STATISTICAL AREA LEVEL IN THE UNITED STATES

**Comment by znmeb** [comment]
I believe the Housing team has Census data - they've been working with Portland State on it. If not, I can point you at some tools for getting it.

**Comment by sky-t** [comment]
@znmeb, thanks for your offer. I actually think we're good on the census data we've cherry picked so far for our purposes, but will reach out if our needs change again.

Team, I've uploaded all the tables I believe we'll need at this point. They are now named as such in the db:
* census_health_insurance
* census_poverty_status_individuals
* census_food_stamps
* census_educational_attainment
* census_geographical_mobility

Once I've updated the data dictionary for these tables, I'll close this ticket.
